test/integration/targets/ansible-test-container/runme.py

```diff
@@ -1068,7 +1034,7 @@ class AptBootstrapper(Bootstrapper):
     @classmethod
     def install_podman(cls) -> bool:
         """Return True if podman will be installed."""
-        return not (os_release.id == 'ubuntu' and os_release.version_id in {'20.04', '22.04'})
+        return not (os_release.id == 'ubuntu' and os_release.version_id == '22.04')
```

Comment:
This can be updated to always return True, since we only test with Ubuntu 24.04 on the controller now.

Reply:
Done.